cmd/podman/containers/restore.go

```diff
@@ -97,20 +99,46 @@ func restore(cmd *cobra.Command, args []string) error {
 	if rootless.IsRootless() {
 		return errors.New("restoring a container requires root")
 	}
-	if restoreOptions.Import == "" && restoreOptions.ImportPrevious != "" {
-		return errors.Errorf("--import-previous can only be used with --import")
+
+	// Find out if this is an image or container ID
```

Comment:
The comment seems misleading to me as we only check if it's an image. At least I cannot spot a check if the input may refer to a container.